scipy/linalg/flapack_pos_def_tri.pyf.src

```diff
@@ -12,3 +12,73 @@ subroutine <prefix>ptsv(n, nrhs, d, e, b, info)
     integer intent(out) :: info
 
 end subroutine <prefix>ptsv
+
+
+subroutine <prefix>pttrf(n, d, e, info)
+    ! d, e, info  = pttrf(d, e, overwrite_d=0, overwrite_e=0)
+    !
+    ! ?PTTRF computes the L*D*L**T factorization of a symmetric positive
+    ! definite tridiagonal matrix A.  The factorization may also be regarded as ! having the form A = U**T*D*U.
+
+    callstatement (*f2py_func)(&n, d, e, &info)
+    callprotoargument int*, <ctypereal>*, <ctype>*, int*
+
+    integer intent(hide), depend(d), check(n > 1) :: n = len(d)
```

Comment:
Because this is connected to `d` you don't need the check since n=0 is allowed in the actual Fortran code as an early exit.

Reply:
removed `check(n > 1)`